From e4e5ee6493c0bd1e397d6c19936ffb395f46cf40 Mon Sep 17 00:00:00 2001
From: Jayanthi Annadurai <jayanthia@netlogicmicro.com>
Date: Sun, 20 Mar 2011 12:58:06 -0700
Subject: [PATCH 191/565] EVP: Adding config files for XLP Eval board

EVP: Adding config files for XLP Eval board

Based on Broadcom SDK 2.3.

Signed-off-by: Jayanthi Annadurai <jayanthia@netlogicmicro.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/mips/Makefile b/arch/mips/Makefile
index ccb61c1..391c975 100644
--- a/arch/mips/Makefile
+++ b/arch/mips/Makefile
@@ -189,6 +189,8 @@ else
 COMPRESSION_FNAME		= vmlinux
 endif
 
+cflags-y			+= -I$(srctree)/arch/mips/include/asm/mach-generic
+
 #
 # Board-dependent options and extra files
 #
@@ -212,8 +214,8 @@ include $(srctree)/arch/mips/Kbuild.platforms
 core-$(CONFIG_NLM_XLP) 		+= arch/mips/netlogic/boot/
 cflags-$(CONFIG_NLM_XLP)	+= -DXLP -I$(srctree)/arch/mips/include/asm/netlogic/hal
 cflags-$(CONFIG_NLM_XLP)	+= -I$(srctree)/arch/mips/netlogic/boot
+load-$(CONFIG_NLM_XLP_BOARD) += $(CONFIG_NLM_COMMON_LOAD_ADDRESS)
 
-cflags-y			+= -I$(srctree)/arch/mips/include/asm/mach-generic
 drivers-$(CONFIG_PCI)		+= arch/mips/pci/
 
 #
-- 
1.8.4.93.g57e4c17

